src/components/Layout/DrawerNavigation.tsx

```diff
@@ -19,6 +18,7 @@ import { useDrawerContext, DrawerButton } from "../../context/drawer";
 import Constants from "expo-constants";
 import { AlertModalContext, CONFIRMATION_MESSAGE } from "../../context/alert";
 import { useTranslate } from "../../hooks/useTranslate/useTranslate";
+import { logout } from "../Logout/LogoutScreen";
```

Comment:
Actually if it’s only being used here, maybe the function should only be in this file instead?